orb/views.py

```diff
@@ -888,16 +888,35 @@ def resource_add_free_text_tags(resource, tag_text, user, category_slug):
     category = Category.objects.get(slug=category_slug)
 
     for tag_name in free_text_tags:
-        tag, created = Tag.tags.get_or_create(name=tag_name, defaults={
-            'category': category,
-            'create_user': user,
-            'update_user': user,
-        })
+        try:
+            tag = Tag.tags.rewrite(False).get(name=tag_name)
```

Comment:
This line here with `rewrite(False)` is the crux of the update. This checks against the `name` database field regardless of what language is actively being used.